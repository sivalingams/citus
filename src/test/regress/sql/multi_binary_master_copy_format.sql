--
-- MULTI_BINARY_MASTER_COPY
--


SET citus.next_shard_id TO 430000;


-- Try binary master copy for different executors

SET citus.binary_master_copy_format TO 'on';

SELECT count(*) FROM lineitem;
SELECT l_shipmode FROM lineitem WHERE l_partkey = 67310 OR l_partkey = 155190;


SELECT count(*) FROM lineitem;
SELECT l_shipmode FROM lineitem WHERE l_partkey = 67310 OR l_partkey = 155190;
