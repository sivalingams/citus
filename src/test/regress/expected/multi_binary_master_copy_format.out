--
-- MULTI_BINARY_MASTER_COPY
--
SET citus.next_shard_id TO 430000;
-- Try binary master copy for different executors
SET citus.binary_master_copy_format TO 'on';
SELECT count(*) FROM lineitem;
 count
---------------------------------------------------------------------
 12000
(1 row)

SELECT l_shipmode FROM lineitem WHERE l_partkey = 67310 OR l_partkey = 155190;
 l_shipmode
---------------------------------------------------------------------
 TRUCK
 MAIL
(2 rows)

SELECT count(*) FROM lineitem;
 count
---------------------------------------------------------------------
 12000
(1 row)

SELECT l_shipmode FROM lineitem WHERE l_partkey = 67310 OR l_partkey = 155190;
 l_shipmode
---------------------------------------------------------------------
 TRUCK
 MAIL
(2 rows)

